internal/exec/stages/disks/luks.go

```diff
@@ -250,7 +251,7 @@ func (s *stage) createLuks(config types.Config) error {
 
 		// open the device
 		if _, err := s.Logger.LogCmd(
-			exec.Command(distro.CryptsetupCmd(), "luksOpen", devAlias, luks.Name, "--key-file", keyFilePath),
+			exec.Command(distro.CryptsetupCmd(), luksOpenArgs(luks, keyFilePath)...),
```

Comment:
How long is the keyfile on the system? does it go away after reboot?

Reply:
Sorry I know this in not super relevant, but I am looking at  this code independant of this MR, and was wondering about how some of these parts work. 





That being said, this does LGTM.

Reply:
If the keyfile was specified by the user, or no particular keying was specified at all (=> we automatically generate a keyfile), then the keyfile is written to `/etc` by the files stage.  If only Clevis is specified, then the keyfile is temporary and is deleted (and the LUKS key slot is cleared) after the Clevis binding has been added.